File: OMKARAUTOAPI/Consignment/Models/LorryHireListModel.cs

using Shared.Models;

namespace Consignment.Models
{
    public class LorryHireListModel
    {
        public List<LorryHireMasterModel> LorryHireList { get; set; }
        public PaginationMetaData PageMetaData { get; set; }
    }

}

File: OMKARAUTOAPP/src/app/models/lorryhirelistmodel.ts
import { Paginationmodel } from "./paginationmodel";
import { Lorryhiremastermodel } from "./lorryhiremastermodel";


export class Lorryhirelistmodel {
    lorryHireList: Lorryhiremastermodel[] = [];
    pageMetaData: Paginationmodel = new Paginationmodel;
}
